device/quanta/x86_64-quanta_ix7_rglbmc-r0/Quanta-IX7-32X/port_config.ini

```diff
@@ -1,33 +1,33 @@
-# name          lanes                 alias             index    speed
-Ethernet0       29,30,31,32           hundredGigE1      1        100000
-Ethernet4       33,34,35,36           hundredGigE2      2        100000
-Ethernet8       41,42,43,44           hundredGigE3      3        100000
-Ethernet12      45,46,47,48           hundredGigE4      4        100000
-Ethernet16      1,2,3,4               hundredGigE5      5        100000
-Ethernet20      5,6,7,8               hundredGigE6      6        100000
-Ethernet24      9,10,11,12            hundredGigE7      7        100000
-Ethernet28      13,14,15,16           hundredGigE8      8        100000
-Ethernet32      17,18,19,20           hundredGigE9      9        100000
-Ethernet36      21,22,23,24           hundredGigE10     10       100000
-Ethernet40      25,26,27,28           hundredGigE11     11       100000
-Ethernet44      37,38,39,40           hundredGigE12     12       100000
-Ethernet48      49,50,51,52           hundredGigE13     13       100000
-Ethernet52      53,54,55,56           hundredGigE14     14       100000
-Ethernet56      57,58,59,60           hundredGigE15     15       100000
-Ethernet60      61,62,63,64           hundredGigE16     16       100000
-Ethernet64      65,66,67,68           hundredGigE17     17       100000
-Ethernet68      69,70,71,72           hundredGigE18     18       100000
-Ethernet72      73,74,75,76           hundredGigE19     19       100000
-Ethernet76      77,78,79,80           hundredGigE20     20       100000
-Ethernet80      93,94,95,96           hundredGigE21     21       100000
-Ethernet84      101,102,103,104       hundredGigE22     22       100000
-Ethernet88      105,106,107,108       hundredGigE23     23       100000
-Ethernet92      109,110,111,112       hundredGigE24     24       100000
-Ethernet96      113,114,115,116       hundredGigE25     25       100000
-Ethernet100     117,118,119,120       hundredGigE26     26       100000
-Ethernet104     121,122,123,124       hundredGigE27     27       100000
-Ethernet108     125,126,127,128       hundredGigE28     28       100000
-Ethernet112     81,82,83,84           hundredGigE29     29       100000
-Ethernet116     85,86,87,88           hundredGigE30     30       100000
-Ethernet120     89,90,91,92           hundredGigE31     31       100000
-Ethernet124     97,98,99,100          hundredGigE32     32       100000
+# name          lanes                 alias             index    speed    valid_speeds
+Ethernet0       29,30,31,32           hundredGigE1      1        100000   100000,40000
```

Comment:
what is valid_speeds? i do not remember we have introduced this concept in port_config.ini.

Reply:
also i do not see if there is parsing logic on this column.

Reply:
These are for Broadcom PDE. I removed them in https://github.com/Azure/sonic-buildimage/pull/6971/commits/04323af41a85dd01553488e7195899eab47ada46